### ISSUE TITLE
Illegal invocation (same as 234)

### ISSUE THREAD

**Issue by rikinshah23** [SupremeTechnopriest/react-idle-timer#238]
I tried with the latest version and it gives me the error.

I needed to use the latest version with modal prompt feature which is necessary for the web app.

I used A minimal example with the bare essential configuration.

`   "react-idle-timer": "^5.0.0-rc.20",`
```

import { useIdleTimer } from 'react-idle-timer'

export const App = () => {
  const onIdle = () => {
    // Close Modal Prompt
    // Do some idle action like log out your user
  }

  const onActive = (event) => {
    // Close Modal Prompt
    // Do some active action
  }

  const idleTimer = useIdleTimer({ onIdle, onActive })

  return (
    <HomePage />
  )
}
```

![image](https://user-images.githubusercontent.com/26806597/161316304-77d8dd85-e98e-4ab8-a01a-84a9825de007.png)

**Comment by SupremeTechnopriest** [comment]
Can you tell me which version of react you are using?

**Comment by rikinshah23** [comment]
> Can you tell me which version of react you are using?

 "react": "^17.0.2",

**Comment by SupremeTechnopriest** [comment]
Confirmed.  I found the issue.  Will fix it today.

**Comment by SupremeTechnopriest** [comment]
This has been resolved in version `5.0.0-rc.21`.  Thanks for catching this.  Worth noting that if you want to use the web worker timers, you now have to explicitly set the timer interface.  More on that [here](https://idletimer.dev/docs/getting-started/new#web-worker-timers).
